hack/run-functests.sh

```diff
@@ -6,9 +6,8 @@ LATENCY_TEST_RUN=${LATENCY_TEST_RUN:-"false"}
 which ginkgo
 if [ $? -ne 0 ]; then
 	echo "Downloading ginkgo tool"
-	# drop -mod=vendor flags, otherwise the installation will fail
-	# because of the package can not be installed under the vendor directory
-    GOFLAGS='' go install github.com/onsi/ginkgo/ginkgo@v1.16.5
+    go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@v2.5.1
```

Comment:
is there a reason to bring back `-mod=mod` ?  in case there is no presence of ginkgo will the installation succeed?